doc/de/Addons.md

```diff
@@ -109,6 +109,7 @@ Dieser Hook kann dafür verwendet werden, alternative Erkennungsfunktionen einzu
 `$data` ist ein Array:
         'text' => Der analysierte Text.
         'detected' => (Eingabe/Ausgabe) Das Array mit den erkannten Sprachen.
```

Comment:
Please mention that it's an associative array with the language code as the keys and the language probability as the values.